src/jaegertracing/SpanContext.h

```diff
@@ -45,6 +45,7 @@ class SpanContext : public opentracing::SpanContext {
         , _spanID(0)
         , _parentID(0)
         , _flags(0)
+        , _b3Headers(false)
```

Comment:
This doesn't belong here. Parsing of B3 headers is the job for codecs (see https://github.com/jaegertracing/jaeger-client-cpp/tree/master/src/jaegertracing/propagation). You can use Jaeger clients in other languages as a reference.